### DISCUSSION TITLE
如何从fork源仓库同步最新更改

### DISCUSSION THREAD

**Discussion by giscus-kuokuo-io** [nagaame/giscus-discussions#15] [Announcements]
# 如何从fork源仓库同步最新更改

本文介绍了如何从源仓库中同步更新到自己fork的仓库中，主要包括添加远端、抓取远端改变、合并等步骤。详细介绍了如何在git中使用这些命令来实现同步。如果你也想从源仓库同步最新的代码到自己的仓库中，本文会是一个非常有用的指南。

https://kuokuo.io/2022/04/07/sync-changed-from-source-repository/

**Comment by nagaame** [comment]
testing
